### ISSUE TITLE
Installing triggers a DEPRECATION warning about "using the legacy 'setup.py install' method"

### ISSUE THREAD

**Issue by jhbuhrman** [Enforcer/pylint_google_style_guide_imports_enforcing#4]
```bash
$ python --version
Python 3.9.5

$ pip --version
pip 22.3.1 from  <some path>/venv39/lib/site-packages/pip (python 3.9)
```

When I install this package, using
```bash
$ pip install pylint-google-style-guide-imports-enforcing==1.3.0
```

, I get the following deprecation warning:
> <tt>&nbsp;&nbsp;DEPRECATION: pylint-google-style-guide-imports-enforcing is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559</tt>

This installation still succeeds (for now), though.

**Comment by Enforcer** [comment]
Huh, thanks for the report. I'm gonna look into this soon

**Comment by jhbuhrman** [comment]
Thanks. Nice plugin  BTW! 👍

**Comment by jhbuhrman** [comment]
@Enforcer - I was intrigued by the phrase "_and the 'wheel' package is not installed._" Now, when I install wheel as per `pip install wheel`, the deprecation warning is gone.

Strangely enough, when I try to reproduce this warning in another virtual environment (this time based on Python 3.10, but perhaps that doesn't even matter), I get the same deprecation warning but on installing another package, not this one.

Nonetheless, ensuring that `wheel` is installed might not be the best solution and some Googling around learned me that this issue is now raised on multiple OSS packages. See for example https://github.com/funnel-io/requtests/issues/2#issuecomment-1304737778 for some additional info.
